scst-store/configuration/use-node-port.hbs.md

```diff
@@ -0,0 +1,23 @@
+# Using NodePort
+
+**Note:** NodePort is only recommended when: the cluster does not support ingress or the cluster does not support `LoadBalancer` type to services.  `NodePort` is not supported for a multi-cluster setup, as certificates cannot be modified.
+
+Port forwarding is needed when using the `NodePort` configuration:
```

Comment:
I think this section leading into the partial feels a little weird to me? It feels like many `:` in a short amount of space